templates/core/index.html

```diff
@@ -1,817 +1,692 @@
 {% extends "core/base_core.html" %}
 
 {% block title %}Ubuntu Core{% endblock %}
-{% block meta_description %}Ubuntu Core is Ubuntu for IoT and embedded environments, optimised for security and reliable ota updates. Run snaps in a high-security confined sandbox with bulletproof upgrades and a private app store.{% endblock meta_description %}
-{% block meta_copydoc %}https://docs.google.com/document/d/1UFpefDRSAcndNBnWp0hDyzqk74aciqhBWJEi05htWmU/edit{% endblock meta_copydoc %}
+{% block meta_description %}Ubuntu Core is Ubuntu for IoT and embedded environments, optimised for security and reliable
+ota updates. Run snaps in a high-security confined sandbox with bulletproof upgrades and a private app store.{% endblock
+meta_description %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1UFpefDRSAcndNBnWp0hDyzqk74aciqhBWJEi05htWmU/edit{% endblock
+meta_copydoc %}
 
+{% block body_class %}is-paper{% endblock body_class %}
 {% block content %}
 
-<section class="p-strip--suru">
-  {% include "shared/_rt-kernel-ga-banner.html" %}
-  <div class="row u-equal-height u-vertically-center">
-    <div class="col-7">
-      <h1>The operating system optimised for IoT and Edge</h1>
-      <p class="p-heading--4 u-sv3">Ubuntu Core is a secure, application-centric IoT OS for embedded devices</p>
-      <p>
-        <a class="p-button" href="/download/iot">Try Ubuntu Core now</a>
-        <a class="p-button--positive" href="/engage/intro-to-ubuntu-core22-webinar">Watch the webinar</a>
-      </p>
-    </div>
-    <div class="col-5 u-align--center u-embedded-media">
-      <iframe title="UC22 video" class="u-embedded-media__element" src="https://www.youtube.com/embed/6NDWqH1SrGs" allowfullscreen=""></iframe>
+<section class="p-strip is-shallow u-no-padding--bottom">
+  <div class="p-section row--50-50">
+    <div class="col">
+      <h1 class="u-no-margin--bottom">Ubuntu Core</h1>
+      <h2 class="p-section--shallow">The embedded Linux OS for devices</h1>
```

Comment:
no p-section classes on headings and other text elements, those should be on parent divs